packages/dotcom-ui-shell/src/components/Shell.tsx

```diff
@@ -77,6 +75,7 @@ function Shell(props: TShellProps) {
         <Content contents={props.contents || props.children} />
         <AppContextEmbed appContext={props.appContext} />
         <FlagsEmbed flags={props.flags} />
+        <LoadFontsEmbed />
```

Comment:
This needs to be in the `<head>`

Reply:
Fixed in [this commit](https://github.com/Financial-Times/dotcom-page-kit/pull/769/commits/db8d9e2fabc5992941898f142f357c63f3649f7b).